winterfell/SETUP_WINTERFELL.md

```diff
@@ -1,58 +1,48 @@
 # Setting up Winterfell
 Winterfell is the server that handles all control logic in the application. This includes authentication, handling user information, it is basically the bulk of the main logistics.
 
-### Install Node
-This project is designed to work with node.js v5. 
+### Install Node Modules
+This project is designed to work with node.js v4.x.
 
 If you haven't already, we'll need to install node. Run the following at the same directory level as _package.json_.
+
 ```
+$ brew install node
 $ npm install
 ```
 
-### Install MongoDb No-SQL Database
-Refer to the [MongoDb installation page](https://docs.mongodb.com/master/installation/) to install.
-We used Homebrew to install Mongo.
-```
-$ brew install mongodb
-```
+### Startup Vagrant
 
-### Install Redis In-Memory DataStore
-We used NPM to install [Redis](http://redis.io/).
 ```
-$ npm install redis
+$ vagrant up
 ```
 
-## Running Winterfell locally
-Make sure a _MongoDb_ instance is started in a separate terminal, we assume the database is on port 27017.
-```
-$ mongod
-```
-or
-```
-$ ./start-mongod.sh
-```
+### Run Tests
 
-Make sure a _Redis_ instance is started in a separate terminal, we assume the database is on port 6379.
-```
-$ redis-server
-```
-or
 ```
-$ ./start-redis.sh
+$ vagrant ssh
```

Comment:
can we have a script in the winterfell directory ./run-test.sh that 
calls vagrant ssh, and then runs the tests without having to change the current directory? so like ./vagrant/winterfell/run-tests.sh
that way we can stay on the winterfell directory without having to go back and forth